<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Carlos Bonifácio</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Boost Half Bridge</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Topologia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Topologia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Vin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tensão de Entrada (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tensão de Saída (V)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Vo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DeltaIin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DeltaIinMax (A)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_DeltaIin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DeltaVo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DeltaVoMax (V)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_DeltaVo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vo_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Potência de Saída (W)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Po">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOptimize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Otimizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSwitchSel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Chaves</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCapSel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Capacitores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDiodeSel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Diodos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDissSel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Dissipadores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCableSel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Cabos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCoreSel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolher Núcleos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConfiguracoesDoOtimizador">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Configurações do Otimizador</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_JMax">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_JMax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>JMax (A/m2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConfiguracoesDoOtimizador_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parâmetros de Otimização</string>
    </property>
   </widget>
   <widget class="MplWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>360</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>280</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIn_cio">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuBiblioteca">
    <property name="title">
     <string>Library</string>
    </property>
    <widget class="QMenu" name="menuAdd_Component">
     <property name="title">
      <string>Add Component</string>
     </property>
     <addaction name="actionSwitch"/>
     <addaction name="actionDiode"/>
     <addaction name="actionCapacitor"/>
     <addaction name="actionCabl"/>
     <addaction name="actionMagnetic_Core"/>
    </widget>
    <addaction name="menuAdd_Component"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOtimizador"/>
   </widget>
   <addaction name="menuIn_cio"/>
   <addaction name="menuBiblioteca"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="text">
    <string>Switch</string>
   </property>
  </action>
  <action name="actionDiode">
   <property name="text">
    <string>Diode</string>
   </property>
  </action>
  <action name="actionCapacitor">
   <property name="text">
    <string>Capacitor</string>
   </property>
  </action>
  <action name="actionCabl">
   <property name="text">
    <string>Cable</string>
   </property>
  </action>
  <action name="actionMagnetic_Core">
   <property name="text">
    <string>Magnetic Core</string>
   </property>
  </action>
  <action name="actionOtimizador">
   <property name="text">
    <string>Otimizador</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
